tests/testthat/test-Simulations-class.R

```diff
@@ -181,3 +181,46 @@ test_that("DualSimulations user constructor arguments names are as expected", {
     ordered = TRUE
   )
 })
+
+# GeneralSimulationsSummary ----
+test_that("GeneralSimulationsSummary generates object correctly", {
+  target_value <- 1
+  target_dose_interval_value <- 2
+  nsim_value <- 3L
+  mean_tox_risk_value <- 4
+  dose_selected_value <- 5
+
+  result <- expect_silent(
+    new("GeneralSimulationsSummary",
```

Comment:
I would use here the internal constructor `.GeneralSimulationsSummary()` instead of `new`